.github/workflows/gateway-conformance.yml

```diff
@@ -85,7 +88,28 @@ jobs:
       # 5. Start the kubo-gateway
       - name: Start kubo-gateway
         run: |
-          ./ipfs daemon --offline &
+          ./ipfs daemon &
```

Comment:
Note: switched away from offline, but switched the profile to `test` since we need the networking capabilities for the p2p-proxy.